docs/api/style/focus-source.md

```diff
@@ -97,12 +97,16 @@ The `handle.current()` method does not accept any arguments and returns one of t
 
 ### handle.used(`<string>`)
 
-The `handle.used()` method accepts one string argument and returns `true` if it matches the current interaction type.
+The `handle.used()` method accepts one string argument and returns `true` if that interaction type has ever been used before.
```

Comment:
actually I was referring to line 77. but this fix is correct, too! speaking of line 77, do you mind adding `handle.unlock()` and its legacy alias `handle.lock(false)` to the Usage block as well?

It's ok as it is, though. I'll have to add another commit for CHANGELOG.md and the Changes block in this document anyway…